app/services/upload_documents_service.rb

```diff
@@ -37,4 +38,12 @@ def attach_files(planning_application, file, file_params)
       document.file.attach(io: file, filename: new_filename(file_params[:filename]).to_s)
     end
   end
+
+  def fetch_file(filename)
+    if @planning_application.from_production?
+      URI.parse(filename).open("api-key" => ENV.fetch("PLANX_FILE_PRODUCTION_API_KEY", nil))
+    else
+      URI.parse(filename).open("api-key" => ENV.fetch("PLANX_FILE_API_KEY", nil))
+    end
+  end
```

Comment:
how about something like that
```
  def fetch_file(filename)
    URI.parse(filename).open("api-key" => planx_file_api_key)
  end

  def planx_file_api_key
    @planning_application.from_production? ? ENV.fetch("PLANX_FILE_PRODUCTION_API_KEY", nil) : ENV.fetch("PLANX_FILE_API_KEY", nil)
  end
```

Reply:
👍 rubocop corrected `planx_file_api_key` to an if/else statement